contrib/gitian/README.md

```diff
@@ -171,6 +195,25 @@ Take a look in the assert files and note the SHA256 checksums listed there.
 
 You should verify that the checksum that is listed matches each of the binaries you actually built.
 This may be done on Linux using the `sha256sum` command or on MacOS using `shasum --algorithm 256` for example.
+An example script to verify the checksums would be:
+
+```bash
+pushd out/${VERSION}
+
+for TARGET in ${TARGET_LIST}; do
+  ASSERT_DIR="../../sigs/${VERSION}-${TARGET}/${GH_USER}"
+  ASSERT_FILE=$(ls "${ASSERT_DIR}")
+  ASSERT_FULL_PATH="${ASSERT_DIR}/${ASSERT_FILE}"
+  if ! sha256sum --ignore-missing -c "${ASSERT_FULL_PATH}" ; then
+    echo "FAILED for ${TARGET} ! Please inspect manually: ${ASSERT_FULL_PATH}"
+  fi
+done
+
```

Comment:
````
for ASSERT in ../../sigs/${VERSION}-*/*/*.assert; do
  if ! sha256sum --ignore-missing -c "${ASSERT_FULL_PATH}" ; then
    echo "FAILED for ${TARGET} ! Please inspect manually: ${ASSERT_FULL_PATH}"
  fi
done
````

The build procedure only creates one assert file per target, there's no need to overspecify these globs. If there are extraneous files then something has gone wrong in the build process.

Reply:
Thanks for the snippet.